client/signup/steps/about/index.jsx

```diff
@@ -353,84 +376,91 @@ class AboutStep extends Component {
 	};
 
 	renderGoalCheckboxes() {
-		const { translate } = this.props;
+		const { translate, siteGoalsArray } = this.props;
+		const shareOption = ( // eslint-disable-line no-unused-vars
```

Comment:
Since all options are almost same, it would be better if you avoid repeating the shared part and return it from the inside of the callback function that is passed to `map`.

```
options.map( item => {
  return (
   <FormLabel htmlFor="promote" className="about__checkbox-option" key={ item.id }>
   ...
     <span className="about__checkbox-label">
       { item.description }
     </span>
   </FormLabel>
 );
} );
```